src/js/actions/layers.js

```diff
@@ -134,6 +132,135 @@ define(function (require, exports) {
      */
     var METADATA_NAMESPACE = global.EXTENSION_DATA_NAMESPACE;
 
+    var multiGetRef = function (docRef, layerRef, properties) {
```

Comment:
At least this and the next declaration should be underscore-prefixed and JSDoc'd with `@private`